CMake/External_Caffe.cmake

```diff
@@ -255,6 +255,13 @@ else()
     )
 endif()
 
+if(fletch_BUILD_CUDA_CSTD98)
+  set( CAFFE_CXXFLAGS_OVERRIDE
+    -DCMAKE_CXX_FLAGS=-std=c++98 )
```

Comment:
The point isn't to specify it for the entire repository, only projects using CUDA.